packages/ckeditor5-dev-tests/bin/check-dependencies.js

```diff
@@ -13,134 +13,164 @@ const glob = require( 'glob' );
 const depCheck = require( 'depcheck' );
 const chalk = require( 'chalk' );
 
-const packageDirectory = process.argv[ 2 ] ? path.resolve( process.argv[ 2 ] ) : process.cwd();
-const packageJson = require( path.join( packageDirectory, 'package.json' ) );
-const nonExistingCSSFiles = [];
-
-const depCheckOptions = {
-	// We need to add all values manually because if we modify it, the rest is being lost.
-	parsers: {
-		'*.css': parsePostCSS,
-		'*.js': depCheck.parser.es6,
-		'*.jsx': depCheck.parser.jsx,
-		'*.ts': depCheck.parser.typescript,
-		'*.vue': depCheck.parser.vue
-	},
-	ignoreDirs: [ 'docs', 'build' ],
-	ignoreMatches: [ 'eslint', 'eslint-plugin-ckeditor5-rules', 'eslint-plugin-mocha', 'husky', 'lint-staged', 'webpack-cli' ]
-};
-
-if ( Array.isArray( packageJson.depcheckIgnore ) ) {
-	depCheckOptions.ignoreMatches.push( ...packageJson.depcheckIgnore );
-}
+const packagePaths = getPackagePaths( process.argv.slice( 2 ) );
```

Comment:
As I can see, there is a breaking change. In the previous version, if you missed the package directory, the CWD was used instead.

Reply:
But our CI will work with this version so we can leave it.